ONSdigital/eq-author-app
# Validation numbers

### What is the context of this PR?
This adds validation numbers to the left hand nav. It does nothing with isNew and so currently shows all errors.

### How to review 
1. Create a questionnaire with validation errors
1. See the error count reflect in the left hand navigation
1. See the changes in validation count updated in real time


### Dependencies
- [x] - https://github.com/ONSdigital/eq-author-terraform/pull/22
